mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -113,3 +113,45 @@ Description of the change and how users should adjust their configuration (if ne
 ### `deletion_policy` now defaults to `DELETE`
 
 Previously, `google_firebase_web_app` deletions default to `ABANDON`, which means to only stop tracking the WebApp in Terraform. The actual app is not deleted from the Firebase project. If you are relying on this behavior, set `deletion_policy` to `ABANDON` explicitly in the new version.
+
+## Resource: `google_firebase_project_location`
```

Comment:
This should come before the previous resource to maintain alphabetical order

Reply:
Done